### PR TITLE
afinfo: add German translation

### PR DESCRIPTION
**Pull request by therealhappy**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.de/osx/afinfo.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# afinfo
+
+> Audiodatei-Metadaten-Parser für OS X.
+> Eingebauter Befehl von OS X.
+> Weitere Informationen: <https://keith.github.io/xcode-man-pages/afinfo.1.html>.
+
+- Anzeige von Informationen zu einer bestimmten Audiodatei:
+
+`afinfo {{pfad/zur/datei}}`
```
Comment (pixelcmtd) [inline pages.de/osx/afinfo.md:9]:
```suggestion
`afinfo {{pfad/zu/datei}}`
```

File: pages.de/osx/afinfo.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# afinfo
+
+> Audiodatei-Metadaten-Parser für OS X.
+> Eingebauter Befehl von OS X.
+> Weitere Informationen: <https://keith.github.io/xcode-man-pages/afinfo.1.html>.
+
+- Anzeige von Informationen zu einer bestimmten Audiodatei:
```
Comment (pixelcmtd) [inline pages.de/osx/afinfo.md:7]:
```suggestion
- Zeige Informationen zu einer bestimmten Audiodatei an:
```

File: pages.de/osx/afinfo.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+`afinfo {{pfad/zur/datei}}`
+
+- Eine einzeilige Beschreibung der Audiodatei ausgeben:
```
Comment (pixelcmtd) [inline pages.de/osx/afinfo.md:11]:
```suggestion
- Gib eine einzeilige Beschreibung der Audiodatei aus:
```

File: pages.de/osx/afinfo.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+`afinfo --brief {{pfad/zur/datei}}`
+
+- Druckt Metadaten und den Inhalt des InfoDictionary der Audiodatei:
```
Comment (pixelcmtd) [inline pages.de/osx/afinfo.md:15]:
```suggestion
- Gib Metadaten und den Inhalt des InfoDictionary der Audiodatei aus:
```

File: pages.de/osx/afinfo.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+`afinfo --info {{pfad/zur/datei}}`
+
+- Druckt die Ausgabe im XML-Format:
```
Comment (pixelcmtd) [inline pages.de/osx/afinfo.md:19]:
```suggestion
- Gib die Daten im XML-Format aus:
```

File: pages.de/osx/afinfo.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+> Eingebauter Befehl von OS X.
+> Weitere Informationen: <https://keith.github.io/xcode-man-pages/afinfo.1.html>.
+
+- Anzeige von Informationen zu einer bestimmten Audiodatei:
+
+`afinfo {{pfad/zur/datei}}`
+
+- Eine einzeilige Beschreibung der Audiodatei ausgeben:
+
+`afinfo --brief {{pfad/zur/datei}}`
```
Comment (pixelcmtd) [inline pages.de/osx/afinfo.md:13]:
```suggestion
`afinfo --brief {{pfad/zu/datei}}`
```

File: pages.de/osx/afinfo.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+- Druckt Metadaten und den Inhalt des InfoDictionary der Audiodatei:
+
+`afinfo --info {{pfad/zur/datei}}`
```
Comment (pixelcmtd) [inline pages.de/osx/afinfo.md:17]:
```suggestion
`afinfo --info {{pfad/zu/datei}}`
```

File: pages.de/osx/afinfo.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+- Druckt die Ausgabe im XML-Format:
+
+`afinfo --xml {{pfad/zur/datei}}`
```
Comment (pixelcmtd) [inline pages.de/osx/afinfo.md:21]:
```suggestion
`afinfo --xml {{pfad/zu/datei}}`
```

File: pages.de/osx/afinfo.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+
+- Warnungen für die Audiodatei ausgeben, falls vorhanden:
```
Comment (pixelcmtd) [inline pages.de/osx/afinfo.md:23]:
```suggestion
- Gib Warnungen für die Audiodatei aus, falls vorhanden:
```

File: pages.de/osx/afinfo.md [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+
+`afinfo --warnings {{pfad/zur/datei}}`
```
Comment (pixelcmtd) [inline pages.de/osx/afinfo.md:25]:
```suggestion
`afinfo --warnings {{pfad/zu/datei}}`
```

File: pages.de/osx/afinfo.md [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+
+- Hilfe anzeigen:
```
Comment (pixelcmtd) [inline pages.de/osx/afinfo.md:27]:
```suggestion
- Zeige eine Hilfe an:
```

File: pages.de/osx/afinfo.md (hunk 1 above)
Comment (therealhappy) [reply pages.de/osx/afinfo.md:9]: Which one is the preferred one? Both exists in other German translations.

Comment (pixelcmtd) [reply pages.de/osx/afinfo.md:9]: Our [Common Arguments](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md) only mention `pfad/zu/` and from what I can remember _zum_/_zur_/etc is informally discouraged (hence much less common), but I don't think we have a rule regarding this either.

Comment (therealhappy) [reply pages.de/osx/afinfo.md:9]: Thanks for the clarification. I might go through the other files and fix them accordingly.

**Review by kbdharun — APPROVED** [review]
LGTM, Welcome to tldr and thanks for your contribution.

**Review by blueskyson — APPROVED** [review]

**Pull request merged**